docs/http_access_API_definition.md

```diff
@@ -4,22 +4,22 @@
 
 ### `admin_online_broker_configure`
 
-Online the configuration of the Brokers which are new or offline. The Topics' configuration will be distributed to relate Brokers as well.
+The online configuration of the Brokers are new or offline. The configuration of Topics are distributed to related Brokers as well.
 
 __Request__
 
 |name|must|description|type|
 |---|---|---|---|
-|brokerId|yes|the id of broker. It supports bulk brokerId which separated by `,`. The maximum <br> number of a bulk is 50. The brokerId should be distinct in case of bulk value  |int|
-|modifyUser| yes|the user who executes this |String|
-|modifyDate| no|the modify date in the format of "yyyyMMddHHmmss"|String|
-|confModAuthToken| yes|the authorization key |String|
+|brokerId|yes|The broker ID. It supports bulk brokerId which is separated by `,`. The maximum number of a bulk is `50`. The brokerId should be distinct in case of bulk value  |int|
```

Comment:
1. Use uppercase letter for the description, since it's a sentence.
2. The description for `modifyUser`, `modifyDate` is not clear. It's the same case for other description.

Reply:
```suggestion
|brokerId|yes|Broker ID(s). It supports multiple broker IDs separated by `,`. The maximum number of IDs is `50`. Broker IDs should be distinct in case of bulk value  |int|
```

I don't quite understand the meaning of "in case of".

Also I don't understand why this parameter can be separated by `,` but the type is `int`. Should it be `String`?

Reply:
When the API call was originally designed, the content passed was relatively short, we carried the parameters in the URL to complete the interaction; later, considering that most of the parameters of some operations are the same, and only brokerid and topicname are different, we separate them by "," characters. According to the above introduction,  the parameters passing are generally of type String, but after receiving by the server, it will check the values according to the defined type, so the interface description is different from the parameter type definition.
What you are talking about is a problem. In other systems, how would this situation be defined and described?
- - - - - 
最初设计API调用时，传递的内容比较短小，我们将参数在URL里携带完成交互，后来考虑到部分操作大部分参数相同，只有brokerID,topicName这些不一样，我们就通过“,”字符来分隔。按照上面的介绍，参数传递整体是String类型，但服务端接收后，会按照定义的类型进行值检查，从而出现了接口上说明与类型定义的不一样。

你说的这个问题是一个问题，在其他系统里，这种情况会如何定义和描述呢？

Reply:
Maybe the type can be `int array` or `int list` and the description marks comma-separated.

Then those words about bulk even can be simplified or removed because array/list type will be super easy for developers to understand it supports multiple IDs.

One more suggestion, adding some example can improve the readability.